src/test/java/org/folio/rest/api/IsbnNormalizationTest.java

```diff
@@ -0,0 +1,173 @@
+package org.folio.rest.api;
+
+import org.folio.rest.support.builders.RequestItemSummary;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import org.folio.rest.support.Response;
+import static org.folio.util.StringUtil.urlEncode;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import org.folio.rest.support.builders.RequestRequestBuilder;
+import java.util.ArrayList;
+import java.util.UUID;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.folio.rest.support.ApiTests;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.awaitility.Awaitility;
+
+public class IsbnNormalizationTest extends ApiTests {
+  private static final String REQUEST_STORAGE_URL = "/request-storage/requests";
+     // Interesting Times has two ISBNs: 0552167541, 978-0-552-16754-3
+
+  @Before
+  public void setUp() throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+	  createRequests();
+	  Awaitility.await()
+      .atMost(10, TimeUnit.SECONDS)
+      .until(requestsLoaded(), is(true));
+  }
+
+  @Test
+  public void searchForNormalizedIsbns() {
+
+    find("isbn = 0-552-16754-1",      "Interesting Times");
+
+    find("isbn = 05-5*",              "Interesting Times");
+
+    find("isbn = 9780552167543",      "Interesting Times");
+
+    find("isbn = 9780*", "Interesting Times", "Temeraire");  
+    
+    find("isbn = 9-7-8-055-2167-543", "Interesting Times");
+
+    find("isbn = 552-16754-3");
+
+    find("isbn = 552*");
```

Comment:
Do each of these examples demonstrate the same behaviour, specific separate behaviour or are there some sub-categories e.g.
* should find single request by normalized ISBN 
* should find multiple requests by normalized ISBN 
* should find no requests for normalized ISBN that does not exist

Reply:
Each one tests the normalization in a slightly different way: search for an isbn with dashes, search for an isbn with truncation, search for a second isbn in an item with two, etc.  The original test code I was using as a model had these all separated out, with method names that indicated exactly what searching behavior was being tested ("canSearchForIsbnsWithDashes," for example).  But as I mentioned in the notes above, I could not get the setup code to run properly in that configuration.  @beforeClass/@beforeAll would silently fail, leaving no records loaded-I *think* this is because those annotations require static methods only, and the setup relies on a whole web of methods that are non-static, many of which are in external files (like RequestItemSummary).  I tried looking up some different methods of getting @Before notations to operate in non-static contexts, but they all either crashed the build or seemed to do nothing.  So I finally just put all the searching tests into a single test and used @before to do the setup-this is not ideal from a readability standpoint, but it was the only way I could figure out to get the test to actually run with the necessary setup.

Reply:
> But as I mentioned in the notes above, I could not get the setup code to run properly in that configuration.

That's what I'm hoping to be able to do.

Please could you put a comment on each of the find invocations describing the intent like you had for the names when they were individual tests. That could help guide the refactoring I'm trying to do.

Thank you for trying a bunch of approaches. I understand none of them working was frustrating, it can definitely be difficult to set up these tests. The test suite workaround (for slow database setup) I put in place a long time ago has become a hinderance to this in quite a few ways :-/

Reply:
Thank you for being patient with me.  I've annotated the individual tests in the latest commit, and I removed the code from the schema concerning the identifiers index.

Reply:
> Thank you for being patient with me. 

Thank you for being receptive to feedback and for trying to navigate your way through some of the poor code in FOLIO.

> I've annotated the individual tests in the latest commit, and I removed the code from the schema concerning the identifiers index.

Thank you